### DISCUSSION TITLE
How powerful will the machine where the games will be run be?

### DISCUSSION THREAD

**Discussion by ernestoyaquello** [SebLague/Chess-Challenge#228] [Q&A] [answered]
This might be a silly question, but what's the standard scenario we should write our time constraints for? The game won't run equally fast on all machines. For example, what on one device will take 2 seconds, on another might take 20. In fact, time can also vary wildly within the same machine depending on whether you run the program on debug or release.

Are we aiming for the bot to run on an old, slow machine? Or can we assume it will run on a modern, powerful one as a release build?

**Comment by SebLague** [comment, marked as answer]
Hi, sorry for the slow response! Yes it will be run in release mode, and I posted some info about the hardware here: https://github.com/SebLague/Chess-Challenge/issues/381
